### DISCUSSION TITLE
Autobrr -> cross-seed: "Tracker: [Could not parse bencoded data]"

### DISCUSSION THREAD

**Discussion by DoAndroids** [cross-seed/cross-seed#339] [Q&A]
Hi all

I've got Autobrr pushing notifications to the cross-seed daemon, which is working (in the sense that the daemon is processing and then auto-injecting into rtorrent).
However, it keeps resulting in an issue with the torrent stopping with error: "Tracker: [Could not parse bencoded data]"

If I then edit the torrent (using rutorrent), and copy / replace the tracker (same string selected from the text copy), then it re-checks and the torrent continues normally.

Any suggestions on how to debug / investigate further?

Thanks

**Comment by mmgoodnow** [comment]
Is this happening on all trackers?

**Reply by DoAndroids** [reply]
So far I've only experienced it on one (Digitalcore), but my tests were not exhaustive.
I now have a few other trackers to potentially try it with.

**Reply by mmgoodnow** [reply]
If you're injecting into rtorrent but reading from deluge, that's the reason you're getting this error.

**Reply by DoAndroids** [reply]
No, in this scenario it was rtorrent as the source client.

**Comment by kingofzeal** [comment]
I had a similar issue as I recently migrated from qbittorrent to rtorrent. DigitalCore reseeds specifically would result in "Could not parse bencoded data", and other reseeds would just stay in a paused state. Editing the torrent (trackers), without making any changes, and saving would allow the torrent to continue.

I don't believe this is an issue with cross-seed, but rather rtorrent.

I run everything in containers, and I ended up setting the rtorrent data directory to `chmod 777 -R`. Additionally, I found https://github.com/crazy-max/docker-rtorrent-rutorrent/discussions/265 and made the recommended change there (this is also the container image I'm using, YMMV). I don't know which one resolved the issue, but since I did both I haven't had any issues with cross-seed injecting torrents (including DigitalCore, which I just confirmed).

**Comment by zakkarry** [comment]
If you simply copied the .torrent files from the qbittorrent client's BT_Backup folder, these torrent files are not 1:1 their original.

I'm not sure if your goal was to get the torrents from one client to the other, but data based matching would be a much easier method as you've found.

qBittorrent for some reason (I think this may have changed in later versions, although they are still not 1:1) - omitted the tracker entirely from the torrent files stored in BT_Backup. They also changed some other things IIRC.

Given what I'm understanding you as describing this sounds like it was likely the problem...
